web/conf.py

```diff
@@ -176,21 +174,19 @@
         ),
         (
             (
-                # (TELECON_LINK, "Teleconference room (Jitsi Meet)"),
                 (TELECON_CALENDAR, "Calendar"),
                 ("/meetings", "Meetings"),
                 (FACEBOOK, "Facebook"),
                 (TWITTER, "Twitter"),
                 (YOUTUBE_CHANNEL, "YouTube"),
-                # (TELECON_NOTES, "Notes from weekly meetups"),
                 # (RITIEK, "GSoC 2018"),
```

Comment:
```suggestion
```
Since it's a few years after 2018.  I hope this doesn't count as scope creep! :scream_cat: